features/networking/operator.feature

```diff
@@ -204,23 +209,34 @@ Feature: Operator related networking scenarios
   Then the step should succeed 
   And the output should contain:
     | The service "<%= project.name %>/test-service" has been marked as idled |
+  
   Given I have a pod-for-ping in the project
   When I execute on the pod:
-    | /usr/bin/curl | --connect-timeout | 5 | <%= cb.service_ip %>:27017 |
+    | /usr/bin/curl | --connect-timeout | 60 | <%= cb.service_ip %>:27017 |
   Then the step should succeed
   And the output should contain:
     | Hello OpenShift |
-  #Inducing flag disablement here an wait 60 seconds for CNO to update it across the nodes
+  
+  #Inducing flag disablement here an polling loop of 300 seconds for CNO to update it across the nodes by checking keywords in sdn logs
   Given as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with: 
     | {"spec":{"defaultNetwork":{"openshiftSDNConfig":{"enableUnidling" : false}}}} |
   # Cleanup required to move operator config back to normal
   Given I register clean-up steps:
   """
   as admin I successfully merge patch resource "networks.operator.openshift.io/cluster" with: 
     | {"spec":{"defaultNetwork":{"openshiftSDNConfig": null}}} |
-  120 seconds have passed
   """
-  And 120 seconds have passed
+  And I wait up to 300 seconds for the steps to pass:
+    """
+    When I run the :logs admin command with:
+      | resource_name | <%= cb.sdn_pod %> |
+      | namespace     | openshift-sdn     |
```

Comment:
@anuragthehatter Thanks for fixing issue.  I saw you add one step to get the sdn or ovn pod in line 202.  but here the namespace is 'openshift-sdn'. So if ovn the ns should be 'openshift-ovn-kubernetes'

Reply:
@zhaozhanqi As we can see, this step is reusable in future irrespective of networktype. Also this particular case is specific to openshift-sdn. This feature won't be supported on OVN as per talk with Casey during this story. So we can keep as is

Reply:
@anuragthehatter thanks. then it's ok